tests/test_methods_and_attributes.py

```diff
@@ -474,3 +474,13 @@ def test_custom_caster_destruction():
 
     # Make sure we still only have the original object (from ..._no_destroy()) alive:
     assert cstats.alive() == 1
+
+
+def test_overload_order_normal():
```

Comment:
You can combine these two into `test_overload_ordering` (I assume it started that way, given the `// test_overload_ordering` comment in the `.cpp`).  The first one isn't a particularly profound test on its own; it's just testing that the normal behaviour works as expected.